### PR TITLE
Fix gamma intensity calculation

### LINKED ISSUE #1433: Documentation Gamma function - factor 100
In the documentation, it is stated that the method material.gammas() "Returns a vector of gamma rays and intensities in decays/s/atom material". I think we should note here that this value refers to per 100 decays. 

So, for example, when I run this method on the material consisting only out of Cs-137, it will return
`(283.5, 4.2351778792770855e-13)`
`(661.657, 6.214028233215172e-08)`

(the first value is the energy of the gamma in keV, and the second value is the intensity. From the documentation you would think the intensity is given in [decays/s/atom], but actually it is given in [100*decays/s/atom])

When I compare this with calculations with the data from [nucleide.org](http://www.nucleide.org/Laraweb/index.php) for the 661.657 keV peak of Cs-137 I get the following:
Specific activity (Am): 3.213 *10e12 [Bq/g] [decays/g/s]   
The intensity is given with 84.99% (note this is another intensity, it states in how many percent of decays this emission will occur).

Specific activity * intensity * mass-cs-137 / mol  = 6.212239912837907e-10 [decays/s/atom]

Maybe I am doing a mistake here, but I am quite sure, there is an error by a factor of 100. Instead of changing the documentation, we could also change the code directly.

### PR DESCRIPTION
**Pull request by ahnaf-tahmid-chowdhury**
## Description
This PR addresses the issue "Documentation Gamma function - factor 100 #1433". The problem was related to the `material.gammas()` method, which incorrectly calculates the intensity of gamma rays, returning the value in [100*decays/s/atom] instead of the expected [decays/s/atom].

## Motivation and Context
The current `material.gammas()` method in `pyne` misrepresents the intensity of gamma rays, leading to incorrect results when used in calculations involving specific activity. This PR aims to fix the intensity calculation and ensure that the method returns gamma intensities in the correct units.

## Changes
This PR introduces a fix to the `material.gammas()` method in the `material.cpp` file. The code has been updated to divide the raw gamma intensity values by 100 to correct the unit representation. With this fix, the method will now return gamma intensities in [decays/s/atom] as documented.

## Behavior
- Current Behavior: The `material.gammas()` method incorrectly calculates and returns gamma intensities in [100*decays/s/atom].
- New Behavior: After the fix, the method will correctly return gamma intensities in [decays/s/atom].

## Other Information
To verify the correctness of the changes, I ran tests with the isotope Cs-137, using an intensity value of 84.99% obtained from nucleide.org. The results now match the expected values, confirming the correction.

```python
from pyne.material import Material
Material({'Cs-137': .8499}).gammas()

Output: [(283.5, 4.2351778792770854e-15), (661.657, 6.214028233215172e-10)]
```

image

## Changelog file
This PR also includes an update to the `CHANGELOG` file. I created a new entry describing the change and referenced this pull request number for easy tracking.

### CODE REVIEW

File: tests/test_material.py [hunk 1]
```diff
@@ -2212,7 +2201,7 @@ def test_decay_u235_h3():
     obs = mat.decay(365.25 * 24.0 * 3600.0)
     if len(obs) < 4:
         # full decay is not installed
-        raise SkipTest
+        raise pytest.skip("full decay is not installed")
```
Comment (paulromano) [inline tests/test_material.py]: I don't think you need to `raise` when calling `pytest.skip`
```suggestion
        pytest.skip("full decay is not installed")
```

File: tests/test_material.py [hunk 2]
```diff
@@ -1981,83 +1974,81 @@ def test_material_gammas():
...
+    assert mat.xrays() == [
+        (93.35, 7.135646069465136e-18),
+        (89.953, 4.4112564001433475e-18),
+        (105.0, 3.394789326064896e-18),
+        (13.0, 2.9652250902501514e-17),
+        (93.35, 5.2767097753586244e-21),
+        (89.953, 3.2620619831267015e-21),
+        (105.0, 2.510398896994686e-21),
+        (13.0, 3.4433858446293e-17),
+    ]
```
Comment (paulromano) [inline tests/test_material.py:2051]: Is the x-ray data also affected by the factor of 100?

Comment (ahnaf-tahmid-chowdhury) [reply tests/test_material.py:2051]: No. I think it occurred due to the use of black code formatter. The values are same.

Comment (paulromano) [reply tests/test_material.py:2051]: Understood, just wanted to make sure it wasn't an oversight :+1:

**Review by paulromano — APPROVED** [review]

**Pull request merged**